backend/apis/events/add-event.api.js

```diff
@@ -14,11 +14,11 @@ app.post("/api/events", (req, res) => {
     nonEmptyString("eventName"),
     nonEmptyString("eventLocation"),
     validDate("eventDate"),
-    validInteger("totalAttendence")
+    validInteger("totalAttendance")
   );
 
   if (validityErrors.length > 0) {
-    return invalidRequest(res, validityErrors, connection);
```

Comment:
`connection` is not defined. The server was dying whenever there was an invalid request to POST /api/events.